src/mixins/common-mixin.ts

```diff
@@ -1,10 +1,14 @@
 import { Component, Vue } from 'vue-property-decorator'
 import { EntityType, PriorityCode, NrRequestActionCodes } from '@/enums'
+import { EmptyBusinessLookup } from '@/interfaces'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
 import { GetFeatureFlag } from '@/plugins'
 
 @Component({})
 export class CommonMixin extends Vue {
+  // Local properties
+  inProgressBusinessLookup = EmptyBusinessLookup
```

Comment:
This local property has no reason to exist in this mixin. It should be in the search component, where it can be initialized/reset when needed, etc.

Reply:
okay